vueapp/src/components/Player.vue

```diff
@@ -3,6 +3,8 @@
   <template v-slot:prepend>
     <v-icon
       :icon="player.isGuest ? 'mdi-account-box-outline' : 'mdi-account-circle'"
+      :color="player?.level?.color"
```

Comment:
When we edit a Guest and change their level, the icon displays as 'no level' even though the the editor shows the correct level.

And in the member's editor the color indicator is not shown.

Reply:
Within `GuestUpsert.vue` and `MemberUpsert.vue` the player's level isn't being set correctly (maybe something to do with how `v-model` is setting it?).
Printing the player's level to the console shows that it is only updated in `v-select`'s model but not in the store (where it is set to undefined).

Reply:
The `GuestUpsert` selection control is fixed in the latest commit (missing the `return-object` attribute).  The `MemberUpsert` doesn't support changing levels at this time.